### ISSUE TITLE
Standalone.TurnLanes test failing on Linux and OSX

### ISSUE THREAD

**Issue by matkoniecz** [valhalla/valhalla#2636]
- https://app.circleci.com/pipelines/github/valhalla/valhalla/4070/workflows/e4caa755-a3bf-4798-9b81-5bebc67ef1d8/jobs/31434
- https://app.circleci.com/pipelines/github/valhalla/valhalla/4070/workflows/e4caa755-a3bf-4798-9b81-5bebc67ef1d8/jobs/31433
- https://app.circleci.com/pipelines/github/valhalla/valhalla/4070/workflows/e4caa755-a3bf-4798-9b81-5bebc67ef1d8/jobs/31435

```
Running main() from /root/project/third_party/googletest/googletest/src/gtest_main.cc
[==========] Running 4 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 4 tests from Standalone
[ RUN      ] Standalone.TurnLanes
[          ] generating map PBF at test/data/gurka_turn_lanes_1/map.pbf
[          ] building tiles in test/data/gurka_turn_lanes_1
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_1 with waypoints D -> C with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.004491560161113739,"lat":-0.0017966240411624313,"type":"break"},{"lon":0.008983120322227478,"lat":0.0,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_1 with waypoints A -> D with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":0.0,"type":"break"},{"lon":0.004491560161113739,"lat":-0.0017966240411624313,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
/root/project/test/gurka/test_turn_lanes.cc:17: Failure
Expected equality of these values:
  prev_edge->turn_lanes_size()
    Which is: 0
  expected_lanes[i - 1].first
    Which is: 4
Incorrect lane count on node index 1
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_1 with waypoints C -> D with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.008983120322227478,"lat":0.0,"type":"break"},{"lon":0.004491560161113739,"lat":-0.0017966240411624313,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
/root/project/test/gurka/test_turn_lanes.cc:17: Failure
Expected equality of these values:
  prev_edge->turn_lanes_size()
    Which is: 0
  expected_lanes[i - 1].first
    Which is: 4
Incorrect lane count on node index 1
[  FAILED  ] Standalone.TurnLanes (847 ms)
[ RUN      ] Standalone.TurnLanesSplitLane
[          ] generating map PBF at test/data/gurka_turn_lanes_2/map.pbf
[          ] building tiles in test/data/gurka_turn_lanes_2
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_2 with waypoints A -> C with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0002694936119951308,"type":"break"},{"lon":0.0003593248256947845,"lat":0.0,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_2 with waypoints A -> E with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0002694936119951308,"type":"break"},{"lon":0.0008983120205812156,"lat":-0.00017966241284739226,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_2 with waypoints A -> G with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0002694936119951308,"type":"break"},{"lon":0.0008983120205812156,"lat":-0.0003593248256947845,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_2 with waypoints A -> H with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0002694936119951308,"type":"break"},{"lon":0.0003593248256947845,"lat":-0.0005389872239902616,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[       OK ] Standalone.TurnLanesSplitLane (861 ms)
[ RUN      ] Standalone.TurnLanesSharedTurnLane
[          ] generating map PBF at test/data/gurka_turn_lanes_3/map.pbf
[          ] building tiles in test/data/gurka_turn_lanes_3
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_3 with waypoints A -> G with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":0.0,"type":"break"},{"lon":0.001437299302779138,"lat":0.0,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_3 with waypoints A -> C with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":0.0,"type":"break"},{"lon":0.0003593248256947845,"lat":-0.00017966241284739226,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_3 with waypoints A -> F with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":0.0,"type":"break"},{"lon":0.0010779744479805232,"lat":-0.00017966241284739226,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[       OK ] Standalone.TurnLanesSharedTurnLane (703 ms)
[ RUN      ] Standalone.TurnLanesMultiLaneShort
[          ] generating map PBF at test/data/gurka_turn_lanes_4/map.pbf
[          ] building tiles in test/data/gurka_turn_lanes_4
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_4 with waypoints A -> E with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0026949362363666298,"type":"break"},{"lon":0.007186496164649725,"lat":0.0,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_4 with waypoints A -> 3 with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0026949362363666298,"type":"break"},{"lon":0.005389872472733259,"lat":-0.005389872472733259,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_4 with waypoints A -> 4 with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0026949362363666298,"type":"break"},{"lon":0.008983120322227478,"lat":-0.005389872472733259,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_4 with waypoints A -> F with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0026949362363666298,"type":"break"},{"lon":0.007186496164649725,"lat":-0.005389872472733259,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[          ] Routing with mjolnir.tile_dir = test/data/gurka_turn_lanes_4 with waypoints A -> G with costing auto
[          ] Valhalla request is: {"locations":[{"lon":0.0,"lat":-0.0026949362363666298,"type":"break"},{"lon":0.007186496164649725,"lat":-0.008084808476269246,"type":"break"}],"costing":"auto","costing_options":{"auto":{"speed_types":["freeflow","constrained","predicted"]}},"shape_match":"map_snap"}
[       OK ] Standalone.TurnLanesMultiLaneShort (773 ms)
[----------] 4 tests from Standalone (3184 ms total)

[----------] Global test environment tear-down
[==========] 4 tests from 1 test suite ran. (3184 ms total)
[  PASSED  ] 3 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] Standalone.TurnLanes

 1 FAILED TEST
test/gurka/CMakeFiles/run-gurka_turn_lanes.dir/build.make:60: recipe for target 'test/gurka/gurka_turn_lanes.log' failed
make[3]: *** [test/gurka/gurka_turn_lanes.log] Error 1
```

**Comment by mloskot** [comment]
@matkoniecz I've edited your issue, as just linking the status badge in README is not a very useful report.

**Comment by danpaz** [comment]
These are failing tests I added in a branch https://github.com/valhalla/valhalla/pull/2635. I'm not sure why the readme badge would be reporting that as a failure as that should be tracking master.

**Comment by mloskot** [comment]
> I'm not sure why the readme badge would be reporting that as a failure as that should be tracking master.

@danpaz Good point. I've fixed it in #2638
